packages/vm/lib/runBlock.ts

```diff
@@ -419,16 +384,31 @@ export async function rewardAccount(
   return account
 }
 
+/**
+ * Returns the encoded tx receipt.
+ */
+export function encodeReceipt(tx: TypedTransaction, receipt: TxReceipt) {
+  const encoded = encode(Object.values(receipt))
+
+  if (!('transactionType' in tx) || tx.transactionType === 0) {
+    return encoded
+  }
+
+  const type = intToBuffer(tx.transactionType)
+  return Buffer.concat([type, encoded])
+}
+
 /**
  * Generates the tx receipt and returns { txReceipt, encodedReceipt, receiptLog }
+ * @deprecated Please use the new `generateTxReceipt` located in runTx.
```

Comment:
Without having taken a look: a just-in-case reminder that these kind of things should be added to the v6 planning issue so that we will not forget.

Reply:
thanks, added